### PR TITLE
feat: common app settings

### LINKED ISSUE #34: Create a common app settings for backend
Common app settings may include:

- JW API endpoints
- Loading from .env.secret (environments, DB_URL, etc.)
- User error messages

### PR DESCRIPTION
**Pull request by j-koziel**
This PR introduces common app settings for the backend

### CODE REVIEW

File: backend/internal/common/jw_api_endpoints.go [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package common
```
Comment (JudeDavis1) [inline backend/internal/common/jw_api_endpoints.go]: Maybe we can just couple the JW API endpoints to our main env secrets file. I think it's better to have them in a dotenv anyway long term.

Maybe we could even just create a general 3rd party API endpoint struct and put it in there? JwAPIEndpoints seems a bit too specific

**Review by JudeDavis1 — CHANGES_REQUESTED** [review]
Lovely stuff.

Even though you haven't used Go that much, you adapted quickly by looking at current implementations. Good strategy to employ, especially when making changes to code that's new to you. Well done! Just left a comment

File: backend/internal/common/jw_api_endpoints.go (hunk 1 above)
Comment (j-koziel) [reply backend/internal/common/jw_api_endpoints.go]: Yeah I agree

Comment (j-koziel) [reply backend/internal/common/jw_api_endpoints.go]: done in https://github.com/cm-core-team/cm-core/pull/76/commits/ae4fd8e8fe2421765a7a6dc34678fe745c541843

File: backend/internal/common/env_secrets.go [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	Environment     string
+	JwAPIEndpoint   string
+}
+
+var EnvSecretsInstance = EnvSecrets{
+	DbUrl:           os.Getenv("DB_URL"),
+	CorsAllowOrigin: os.Getenv("CORS_ALLOW_ORIGIN"),
+	JwtSecret:       os.Getenv("JWT_SECRET"),
+	Environment:     os.Getenv("ENVIRONMENT"),
+	JwAPIEndpoint:   os.Getenv("JW_API_ENDPOINT"),
```
Comment (JudeDavis1) [inline backend/internal/common/env_secrets.go]: Name this something more specific like `GetWeeklyMeetings_JwAPI` or something

**Comment by j-koziel** [comment]
not sure why CI is failing now 🤔

**Review by JudeDavis1 — CHANGES_REQUESTED** [review]
Last thing!

Also could you update the dot env example in the backend so that it's up to date

**Comment by j-koziel** [comment]
For some reason it's not reading the env var for the endpoint... in CI and locally

File: .github/workflows/backend-unit-tests.yml [hunk 3]
```diff
@@ -3,6 +3,12 @@ name: Backend Unit Tests
 on:
   push:
 
+env:
+  JWT_SECRET: ${{vars.JWT_SECRET}}
+  CORS_ALLOW_ORIGIN: ${{vars.CORS_ALLOW_ORIGIN}}
+  DB_URL: ${{vars.DB_URL}}
+  ENVIRONMENT: ${{vars.ENVIRONMENT}}
+  GET_WEEKLY_MEETINGS_JW_API: ${{vars.GET_WEEKLY_MEETINGS_JW_API}}
```
Comment (JudeDavis1) [inline .github/workflows/backend-unit-tests.yml:11]: Well done!

File: backend/cmd/main.go [hunk 4]
```diff
@@ -26,11 +26,14 @@ func main() {
 	}
 
 	fmt.Print("CORS: ")
-	fmt.Println(os.Getenv("CORS_ALLOW_ORIGIN"))
+	fmt.Println(envSecrets.CorsAllowOrigin)
+
+	fmt.Print("JW API ENDPOINT")
+	fmt.Println(envSecrets.GetWeeklyMeetings_JwAPI)
```
Comment (JudeDavis1) [inline backend/cmd/main.go]: Can be removed

File: backend/internal/common/env_secrets.go [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+// This broke
+// var EnvSecretsInstance = EnvSecrets{
+// 	DbUrl:                   os.Getenv("DB_URL"),
+// 	CorsAllowOrigin:         os.Getenv("CORS_ALLOW_ORIGIN"),
+// 	JwtSecret:               os.Getenv("JWT_SECRET"),
+// 	Environment:             os.Getenv("ENVIRONMENT"),
+// 	GetWeeklyMeetings_JwAPI: os.Getenv("GET_WEEKLY_MEETINGS_JW_API"),
+// }
```
Comment (JudeDavis1) [inline backend/internal/common/env_secrets.go]: Also can be removed

**Review by JudeDavis1 — APPROVED** [review]
LGTM great work!

**Pull request merged**